README.md

```diff
@@ -50,5 +50,5 @@ yarn test:libraries
 Generate pdf file with pkcs#1 schema. This action will create pdf file name `signed.pdf` signed by `certificate.cer` in `packages/anon-aadhaar-pcd/build` folder.
 
 ```bash
-yarn pdf:pcd
+yarn gen-proof:pcd
```

Comment:
I think this should be pdf:pcd

Reply:
Should add instruction to gen proof? :D

Reply:
Yes sorry I mixed things up.
I fixed the typo 
Thanks :)